docs/JenkinsBuilder-download-report.py

```diff
@@ -41,35 +40,35 @@
 data = json.loads(data)
 
 # Prepare output
-output='Downloads for %s:%s' % (os.environ['GITHUB_REPO'],'\\n')
+output = 'Downloads for %s:%s' % (os.environ['GITHUB_REPO'], '\\n')
 # Walk releases ... if you do /release/latest you have to discard this for loop and change the next to use data instead of releases :)
 for release in data:
-  for key,values in release.iteritems():
-    # We are only interested in the assets
-    if key == 'assets':
-      first=0
-      for assets in values:
-        name='n/a\n'
-        downloads=0
-        # Fetch the name and downloadcount
-        for asset,details in assets.iteritems():
-          if asset == 'name':
-            file_name=details
-            if first == 0:
-              first=1
-              output=output+ ' - %s:%s' % (file_name[:7],'\\n')
-          if asset == 'download_count':
-            downloads=details
-        # Summarize to output
-        if downloads &gt; 0:
-          output=output+'   %s: %s%s' % (file_name[9:],downloads,'\\n')
+    for key, values in release.items():
+        # We are only interested in the assets
+        if key == 'assets':
+            first = 0
+            for assets in values:
+                name = 'n/a\n'
+                downloads = 0
+                # Fetch the name and downloadcount
+                for asset, details in assets.items():
+                    if asset == 'name':
+                        file_name = details
+                        if first == 0:
+                            first = 1
+                            output += ' - %s:%s' % (file_name[:7], '\\n')
+                    if asset == 'download_count':
+                        downloads = details
+                # Summarize to output
+                if downloads & gt; 0:
```

Comment:
This is a syntax error.  Before and after I edited this file.  I am not sure what it supposed to do.  I am thinking someone copied html into the file and got the escaped `>` now that I am looking at it... 

hmmm...